scout/server/blueprints/variants/controllers.py

```diff
@@ -1585,7 +1578,13 @@ def update_form_hgnc_symbols(store, case_obj, form):
 
     hgnc_symbols = []
     not_found_ids = []
-    genome_build = "38" if case_obj and "38" in str(case_obj.get("genome_build", "37")) else "37"
+    genome_build = None
+    case_obj = case_obj or {}
+
+    for build in ["37", "38"]:
+        if build in str(case_obj.get("genome_build", "37")):
+            genome_build = build
+            break
 
     # retrieve current symbols from form
     if form.hgnc_symbols.data:
```

Comment:
SonarcCloud was complaining about the complexity of this function before, but I've replaced 2 elses with continue and now it's fine (below 15, SonarCloud happy). I'd rather focus on the bugfix than on refactoring. Since it's not super long I'd rather keep it this way, I find it more readable.